test/routes.js

```diff
@@ -1,5 +1,7 @@
 const test = require('ava')
 
+// Do not start application when importing it
+process.env.NO_START = true
```

Comment:
Check out [this commit](https://github.com/AdamVig/stockpile-api/pull/49/commits/4bc2dfa244977a15decc60a84bf3e2989b4b490a). It is an environment variable that I can modify at import time in order to import `index.js` without running the application. Without this, it is impossible to write more than one test using `index.js` because the second time it is imported, there will be a nasty `EADDRINUSE` error (port conflict).